queue_services/entity-pay/src/entity_pay/worker.py

```diff
@@ -70,9 +70,7 @@ async def publish_filing(filing: Filing):
 
 async def publish_email_message(filing: Filing):
     """Publish the email message onto the NATS emailer subject."""
-    option = 'future' if filing.effective_date >= datetime.datetime.utcnow().replace(tzinfo=datetime.timezone.utc)\
-        else 'immediate'
-    payload = create_email_msg(filing.id, filing.filing_type, option)
+    payload = create_email_msg(filing.id, filing.filing_type, 'filed')
```

Comment:
👍

Reply:
Except we might want to change the filename filter depending on what we decide for the filename pattern.
For example, it makes sense to me to specify the full filename here (possibly with a switch statement to handle different entity types).

Reply:
Future...